cli/script/command-parser.ts

```diff
@@ -273,7 +273,11 @@ var argv = yargs.usage(USAGE_PREFIX + " <command>")
             .demand(/*count*/ 1, /*max*/ 2)  // Require one non-optional and one optional argument.
             .example("login", "Logs in to the CodePush server")
             .example("login --accessKey mykey", "Logs in on behalf of the user who owns and created the access key \"mykey\"")
+            .example("login --proxy proxyURL", "Logs in with the specified proxy url")
+            .example("login --noProxy", "Logs in without any proxy set in session file or environment vars")
             .option("accessKey", { alias: "key", default: null, demand: false, description: " Access key to authenticate against the CodePush server with, instead of providing your username and password credentials", type: "string" })
+            .option("proxy", { default: null, demand: false, description: " Proxy url", type: "string" })
+            .option("noProxy", { default: false, demand: false, description: " Suppress any proxy set in session file or environment vars", type: "boolean" })
```

Comment:
that's ok, sorry for my poor english. anything like these description and comments, if not much proper, please let me know